MapboxGeocoder/MBGeocoder.swift

```diff
@@ -239,23 +239,23 @@ public class Geocoder: NSObject {
             
             let apiMessage = json["message"] as? String
             guard data != nil && error == nil && apiMessage == nil else {
-                let apiError = Geocoder.descriptiveError(json, response: response, underlyingError: error)
+                let apiError = Geocoder.descriptiveError(json, response: response, underlyingError: error as NSError?)
                 DispatchQueue.main.async {
-                    errorHandler(error: apiError)
+                    errorHandler(apiError)
                 }
                 return
             }
             
             DispatchQueue.main.async {
-                completionHandler(json: json)
+                completionHandler(json as AnyObject)
```

Comment:
We should change the completion handler to accept an `Any`, to avoid these casts.